libs/failsafe-controller/src/main/java/net/jodah/failsafe/FailsafeController.java

```diff
@@ -103,9 +124,17 @@ public String getId() {
    *
    * @param circuitBreaker the circuit breaker to use
    * @throws NullPointerException if {@code circuitBreaker} is null
+   * @throws IllegalStateException if any of the <code>retry()</code> methods have been called
+   *     already
    */
-  public SyncFailsafe<R> with(CircuitBreaker circuitBreaker) {
-    return new SyncBuilder(circuitBreaker);
+  @SuppressWarnings(
+      "squid:CommentedOutCodeLine" /* currently not supported, will be uncommented once we support it */)
```

Comment:
This will only be implemented when required. At the moment there is no need for it.
Since this is a test library, I see no need to create a ticket.
I would also prefer to leave the commented out code since it is exactly what would be required the day we support it.